docs/quest_template.md

```diff
@@ -8,73 +8,219 @@ Contains all basic definitions of available quests.
+| [ObjectiveText2][102]           | text         |           |     | YES  |         |       |         |
+| [ObjectiveText3][103]           | text         |           |     | YES  |         |       |         |
+| [ObjectiveText4][104]           | text         |           |     | YES  |         |       |         |
+| [VerifiedBuild][105]            | smallint(6)  |           |     | YES  | 0       |       |         |
+
+[1]: #ID
+[2]: #QuestType
+[3]: #QuestLevel
+[4]: #MinLevel
+[5]: #QuestSortID
+[6]: #QuestInfoID
+[7]: #SuggestedGroupNum
+[8]: #RequiredFactionId1
+[9]: #RequiredFactionId2
+[10]: #RequiredFactionValue1
+[11]: #RequiredFactionValue2
+[12]: #RewardNextQuest
+[13]: #RewardXPDifficulty
+[14]: #RewardMoney
+[15]: #RewardBonusMoney
+[16]: #RewardDisplaySpell
+[17]: #RewardSpell
+[18]: #RewardHonor
+[19]: #RewardKillHonor
+[20]: #StartItem
+[21]: #Flags
+[22]: #RequiredPlayerKills
+[23]: #RewardItem1
+[24]: #RewardAmount1
+[25]: #RewardItem2
+[26]: #RewardAmount2
+[27]: #RewardItem3
+[28]: #RewardAmount3
+[29]: #RewardItem4
+[30]: #RewardAmount4
+[31]: #ItemDrop1
+[32]: #ItemDropQuantity1
+[33]: #ItemDrop2
+[34]: #ItemDropQuantity2
+[35]: #ItemDrop3
+[36]: #ItemDropQuantity3
+[37]: #ItemDrop4
+[38]: #ItemDropQuantity4
+[39]: #RewardChoiceItemID1
+[40]: #RewardChoiceItemQuantity1
+[41]: #RewardChoiceItemID2
+[42]: #RewardChoiceItemQuantity2
+[43]: #RewardChoiceItemID3
+[44]: #RewardChoiceItemQuantity3
+[45]: #RewardChoiceItemID4
+[46]: #RewardChoiceItemQuantity4
+[47]: #RewardChoiceItemID5
+[48]: #RewardChoiceItemQuantity5
+[49]: #RewardChoiceItemID6
+[50]: #RewardChoiceItemQuantity6
+[51]: #POIContinent
+[52]: #POIx
+[53]: #POIy
+[54]: #POIPriority
+[55]: #RewardTitle
+[56]: #RewardTalents
+[57]: #RewardArenaPoints
+[58]: #RewardFactionID1
+[59]: #RewardFactionValue1
+[60]: #RewardFactionOverride1
+[61]: #RewardFactionID2
+[62]: #RewardFactionValue2
+[63]: #RewardFactionOverride2
+[64]: #RewardFactionID3
+[65]: #RewardFactionValue3
+[66]: #RewardFactionOverride3
+[67]: #RewardFactionID4
+[68]: #RewardFactionValue4
+[69]: #RewardFactionOverride4
+[70]: #RewardFactionID5
+[71]: #RewardFactionValue5
+[72]: #RewardFactionOverride5
+[73]: #TimeAllowed
+[74]: #AllowableRaces
+[75]: #LogTitle
+[76]: #LogDescription
+[77]: #QuestDescription
+[78]: #AreaDescription
+[79]: #QuestCompletionLog
+[80]: #RequiredNpcOrGo1
+[81]: #RequiredNpcOrGo2
+[82]: #RequiredNpcOrGo3
+[83]: #RequiredNpcOrGo4
+[84]: #RequiredNpcOrGoCount1
+[85]: #RequiredNpcOrGoCount2
+[86]: #RequiredNpcOrGoCount3
+[87]: #RequiredNpcOrGoCount4
+[88]: #RequiredItemId1
+[89]: #RequiredItemId2
+[90]: #RequiredItemId3
+[91]: #RequiredItemId4
+[92]: #RequiredItemId5
+[93]: #RequiredItemId6
+[94]: #RequiredItemCount1
+[95]: #RequiredItemCount2
+[96]: #RequiredItemCount3
+[97]: #RequiredItemCount4
+[98]: #RequiredItemCount5
+[99]: #RequiredItemCount6
+[100]: #Unknown0
+[101]: #ObjectiveText1
+[102]: #ObjectiveText2
+[103]: #ObjectiveText3
+[104]: #ObjectiveText4
+[105]: #VerifiedBuild
```

Comment:
Need to be lower case for the links to work. (But only in the list)

Reply:
Corrected.